src/Controllers/LoggerController.js

```diff
@@ -4,6 +4,8 @@ import AdaptableController from './AdaptableController';
 import { LoggerAdapter } from '../Adapters/Logger/LoggerAdapter';
 
 const MILLISECONDS_IN_A_DAY = 24 * 60 * 60 * 1000;
+const LOG_STRING_TRUCATE_LENGTH = 1000;
```

Comment:
That's fine although now it's not possible to write a custom LoggerAdapter without truncation. We could add a clean() function to the adapter and call from the controller to enable this, instead of doing the cleaning in controller.